x/stake/client/cli/query.go

```diff
@@ -142,44 +131,42 @@ func GetCmdQueryDelegatorBond(storeName string, cdc *wire.Codec) *cobra.Command
 	return cmd
 }
 
-//// get the command to query all the candidates bonded to a delegator
-//func GetCmdQueryDelegatorBonds(storeName string, cdc *wire.Codec) *cobra.Command {
-//cmd := &cobra.Command{
-//Use:   "delegator-candidates",
-//Short: "Query all delegators bond's candidate-addresses based on delegator-address",
-//RunE: func(cmd *cobra.Command, args []string) error {
-
-//bz, err := hex.DecodeString(viper.GetString(FlagAddressDelegator))
-//if err != nil {
-//return err
-//}
-//delegator := crypto.Address(bz)
-
-//key := stake.GetDelegatorBondsKey(delegator, cdc)
-
-//ctx := context.NewCoreContextFromViper()
-
-//res, err := ctx.Query(key, storeName)
-//if err != nil {
-//return err
-//}
-
-//// parse out the candidates list
-//var candidates []crypto.PubKey
-//err = cdc.UnmarshalBinary(res, candidates)
-//if err != nil {
-//return err
-//}
-//output, err := wire.MarshalJSONIndent(cdc, candidates)
-//if err != nil {
-//return err
-//}
-//fmt.Println(string(output))
-//return nil
-
-//// TODO output with proofs / machine parseable etc.
-//},
-//}
-//cmd.Flags().AddFlagSet(fsDelegator)
-//return cmd
-//}
+// get the command to query all the candidates bonded to a delegator
+func GetCmdQueryDelegatorBonds(storeName string, cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delegator-candidates",
```

Comment:
Same as above, maybe limit-offset (not sure if necessary for bonds) and clarity on sort order.

Reply:
see response above